quinn-proto/src/token_log.rs

```diff
@@ -0,0 +1,41 @@
+//! Limiting clients' ability to reuse tokens from NEW_TOKEN frames
```

Comment:
I think we should keep all of this in the `token` module, which is still pretty small.

Reply:
Done